README.md

```diff
@@ -26,79 +26,87 @@
 ## Why?
 
 I commonly make quick demo pages or websites with simple content. For these, I don't want to spend time styling them but don't like the ugliness of the default styles.
+
 Water.css is a CSS framework that doesn't require any classes. You just include it in your `<head>` and forget about it, while it silently makes everything nicer.
 
 ## Who?
 
 You might want to use Water.css if you're making a simple static page or demo website that you don't want to spend time styling.
 
-You probably don't want to use it for a production app or something that is more than a simple document. Rule of thumb: if your site has a navbar, don't use Water.css. It's just not meant for that kind of content.
+Although it originally wasn't built for more complex websites, many developers have used Water.css as a base stylesheet and creatively applied custom styles to build out an entire app. Nothing is stopping you from doing the same!
 
 ## How?
 
 Just stick this in your `<head>`:
 
+### 🌙/☀ Automatic Theme:
+
+`<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/dist/water.min.css">`
+
 ### 🌙 Dark Theme:
 
-`<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/dark.min.css">`
+`<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/dist/dark.min.css">`
 
 ### ☀ Light Theme:
 
-`<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/light.min.css">`
+`<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/dist/light.min.css">`
 
 <br>
 
 ### Other options:
 
-> ⚡ An interactive version selection is available [on the **demo page**!](https://watercss.netlify.com/#installation)
+> ⚡ An interactive version selection is available [on the **demo page**!](https://watercss.kognise.dev/#installation)
 
 #### Enforce a theme and ignore `(prefers-color-scheme)`
 
-For the main versions, `dark` or `light` is only treated as a _default theme_: if a user has a system-wide preference for either dark or light mode on their device, `water.css` will respect this. If you want to avoid this behavior and enforce dark or light theme, append `.standalone` to the theme prefix, e.g. `dark.standalone.min.css`.
+For the main `water.css` file, light is only treated as a _default theme_: if a user has a preference for either dark or light mode on their device, the stylesheet will respect this. This detection is made possible through a CSS media query called `prefers-color-scheme`. If you want to avoid this behavior and enforce a theme, use either `dark.css` or `light.css`.
```

Comment:
Given that `prefers-color-scheme` has broad browser support now, I'd reverse the order on this: first mention that `water.css` automatically matches a user's color theme, then note that it defaults to light if the preference can't be detected.

Also, this entire section needs a re-do imo. "Auto" is now a separate third theme instead of being controlled via a "Enfore theme" option and there aren't any "other options" in general now. I'll create a PR. :)

Reply:
Oh, and also:

If we keep light mode as the default/fallback, I'd switch the theme emoji for the Automatic Theme (so it's `☀/🌙`) and also the order of "Light Theme" and "Dark Theme", so Light comes first?

Reply:
> I'd switch the theme emoji for the Automatic Theme (so it's ☀/🌙) and also the order of "Light Theme" and "Dark Theme", so Light comes first?

I considered that but imo the current order looks aesthetically better (lol) and it doesn't really clarify anything that much.

Reply:
Alrighty, that's reason enough for me lol 😄